scripts/pre_commit_linter.py

```diff
@@ -1197,6 +1197,53 @@ def _check_directive_scope(self):
 
         return summary_messages
 
+    def _check_js_file_overview(self):
+        """This function checks the JS files for the presence of their
+        respective file overviews.
+        """
+        if self.verbose_mode_enabled:
+            print 'Starting JS file overview check'
+            print '----------------------------------------'
+
+        summary_messages = []
+        failed = False
+        files_to_check = [
+            filepath for filepath in self.all_filepaths if (
+                filepath.endswith('.js'))
+            and not any(fnmatch.fnmatch(filepath, pattern) for pattern in
+                        EXCLUDED_PATHS)]
+
+        with _redirect_stdout(_TARGET_STDOUT):
+            for filepath in files_to_check:
+                parsed_script = self.parsed_js_files[filepath]
+                file_overview_found = False
+                for comment in parsed_script.comments:
+                    if '@fileoverview' in comment.value:
+                        file_overview_found = True
+                        if comment.type != 'Block':
+                            failed = True
+                            print ('%s --> File overview must be in block '
+                                'comments.' % filepath)
+                if not file_overview_found:
+                    failed = True
+                    print '%s --> Missing file overview.' % filepath
```

Comment:
Please improve the error message as 'Please ensure that file %s should contain a file overview i.e. a short description of the file'.

Reply:
Done.